browser/widevine/brave_widevine_bundle_manager_unittest.cc

```diff
@@ -54,16 +52,18 @@ class TestClient : public content::TestContentClient {
 
 class BraveWidevineBundleManagerTest : public testing::Test {
  public:
-  BraveWidevineBundleManagerTest()
-      : testing_profile_manager_(TestingBrowserProcess::GetGlobal()) {
-  }
+  BraveWidevineBundleManagerTest() {}
   ~BraveWidevineBundleManagerTest() override {}
 
  protected:
   void SetUp() override {
-    ASSERT_TRUE(temp_dir_.CreateUniqueTempDir());
-    ASSERT_TRUE(testing_profile_manager_.SetUp(temp_dir_.GetPath()));
     manager_.is_test_ = true;
+    RegisterWidevineLocalstatePrefs(local_state_.registry());
```

Comment:
Could we call `RegisterLocalState` of `chrome/browser/prefs/browser_prefs.cc`?
It is more general and emulate the normal local state registration process. And in any cases that we might want to register more prefs from different files, then we don't need to explicitly register all of them one by one.

Reply:
Changed to use `RegisterLocalState`.
I used `RegisterWidevineLocalstatePrefs` because this unittest is widevine specific unittest.